src/config.c

```diff
@@ -2719,6 +2701,15 @@ standardConfig configs[] = {
     createStringConfig("tls-ciphersuites", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.tls_ctx_config.ciphersuites, NULL, NULL, updateTlsCfg),
 #endif
 
+    /* Special configs */
+    createSpecialConfig("dir", NULL, MODIFIABLE_CONFIG, setConfigDirOption, getConfigDirOption, rewriteConfigDirOption),
+    createSpecialConfig("watchdog-period", NULL, MODIFIABLE_CONFIG, setConfigWatchdogPeriodOption, getConfigWatchdogPeriodOption, NULL),
```

Comment:
Seems like a bug that the watchdog doesn't support rewrite?

Reply:
@yoav-steinberg can you please edit the top comment to list all the behavior changes in this PR (e.g. conf file and rewrite for watchdog-period), and possibly others?

Reply:
Updated. I hope I didn't miss anything.